server/lib/transforms/insert-ad.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const adMarkup = () => `<div class="ad-container"><amp-ad width="300" height="250" type="doubleclick"></amp-ad></div>`;
+
+module.exports = function insertAd($, options) {
+		const paragraph = $('p').eq(3);
```

Comment:
This may need to be more complicated (see the logic [here](https://github.com/Financial-Times/next-article/blob/master/server/transforms/inline-ad.js#L4)). Depends what other body components there are in AMP, but presumably it's the same as Next?